src/scripts/modules/header/header-template.html

```diff
@@ -39,9 +39,9 @@
       {{! Collect the nav links, forms, and other content for toggling }}
       <div class="collapse navbar-collapse" id="page-nav">
         <ul class="nav nav-justified">
-          <li {{#is page 'browse'}}class="active"{{/is}}><a href="/browse" data-l10n-id="all-navbar-search"></a></li>
-          <li {{#is page 'about'}}class="active"{{/is}}><a href="/about" data-l10n-id="all-navbar-about-us"></a></li>
-          <li {{#is page 'donate'}}class="active"{{/is}}><a href="/donate" data-l10n-id="all-navbar-give"></a></li>
+          <li class="navbar-browse{{#is page 'browse'}} active{{/is}}"><a href="/browse" data-l10n-id="all-navbar-search"></a></li>
```

Comment:
Could we use `id="..."` instead of a class attribute?
Class names have a couple of "unique" features compared to other attributes like `id="..."` and `data-foo=".."`:

- they are usually used for

Reply:
Also, since this attribute is only for automation, would it make sense to prefix the `id` with `automation-` or something so devs know it doesn't interfere with the app or any styling in the CSS?

Reply:
It would be nice it it stayed more relevant to the element that we need to select. The locators look like the following:

```
_root_locator = (By.ID, 'header')
_nav_locator = (By.ID, 'page-nav')
_search_input_locator = (By.ID, 'find_content_input')
```
If this sort of change is too complicated i'll just use css selectors and nth children.

Reply:
I appreciate the desire to make these tests less fragile. nth children is a red-flag for me for that sort of thing. As to the specific changes nav items should be unique-on-the-page, so `id`s sound right to me, as well.  The 'all-navbar-about-us'  thing is used for localization - its the key into the locale dictionaries. 

As to naming, having a policy that "every page structure element that causes navigation when clicked on should have an id starting with `nav-`" seems reasonable to me, without being to verbose. @philschatz how's that sound?

Reply:
Could just the link text be used (rather than creating an attribute or using `:nth`)? That would also help verify that the internationalization code ran. `nav-` seems to make sense if end up using the `id="..."` attribute.